src/main/kotlin/no/nav/klage/oppgave/api/view/kabin/KabinResponse.kt

```diff
@@ -89,4 +89,32 @@ data class CreatedKlagebehandlingStatusForKabin(
 data class TildeltSaksbehandler(
     val navIdent: String,
     val navn: String,
-)
\ No newline at end of file
+)
+
+data class KabinPartView(
+    val id: String,
+    val type: BehandlingDetaljerView.IdType,
+    val name: String?,
```

Comment:
Lurer stadig på om `name` kan være `null`. Jeg tror egentlig ikke det, og da tror jeg BE setter noe som "mangler" etc.

Reply:
Sjekket flyten på hvor vi får disse navnene fra. Organisasjon har non-nullable Navn. Jeg sjekket PDL-docen, og der står det at `fornavn` og `etternavn` er tilgjengelig på alle. Gjør derfor denne non-nullable videre innover i flyten.